main.tf

```diff
@@ -16,4 +16,40 @@ terraform {
 provider "aws" {
   profile = "default"
   region = "sa-east-1"
+}
+
+
+resource "aws_s3_bucket" "state" {
+  bucket = "terraform-state-infra-repository"
+  acl = "private"
+  force_destroy = false
+  tags = {
+    "terraform" = ""
+  }
+}
+
+resource "aws_dynamodb_table" "lockState" {
+  name = "terraform-state-locking"
+  hash_key       = "LockID"
+  tags = {
+    "terraform" = ""
+  }
+  attribute {
+    name = "LockID"
+    type = "S"
+  }
+  write_capacity = 5
+  read_capacity = 5
+}
```

Comment:
Does Terraform demand that the read and write capacities must be 5? Just a curiosity I had.

Reply:
No. When create without this options, terraform complains, so I choose the default value.